src/ConfigSpace/forbidden.py

```diff
@@ -475,6 +475,340 @@ def to_dict(self) -> dict[str, Any]:
         }
 
 
+class ForbiddenGreaterThanClause(ForbiddenClause):
+    """A ForbiddenGreaterThanClause.
+
+    It forbids a value from the value range of a hyperparameter to be
+    *greater than* `value`.
+
+    Forbids the value of the hyperparameter *a* to be greater than 2
```

Comment:
The pre-existing forbidden clauses had some example in their docstring, could you please add such examples here, too?